library/model/user.py

```diff
@@ -64,16 +68,16 @@ def return_books(self, books: list[BorrowedBook]):
         else:
             return None
 
-    def get_reading_credits(self, books: list[Book]) -> int:
+    def get_reading_credits(self) -> int:
         reading_credits: int = 0
-        for book in books:
+        for book in self.borrowed_books:
```

Comment:
We think that the reading_ceredits should be granted not only for the borrowed_bookds but for all books.